tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestDtdData.java

```diff
@@ -304,11 +306,13 @@ private void checkEmpty(
                 // if no children left, treat like EMPTY
                 if (children.isEmpty()) {
                     if (valueAttributes.isEmpty()) {
-                        errln(
-                                type
-                                        + "\t|| "
-                                        + showPath(parents)
-                                        + "||path has neither value NOR value attributes||");
+                        if (!ALLOWED_EMPTY_NO_VALUE_PATHS.contains(showPath(parents))) {
+                            errln(
+                                    type
+                                            + "\t|| "
+                                            + showPath(parents)
+                                            + "||DTD has neither value NOR value attributes (only special or deprecated children)||");
+                        }
```

Comment:
This and following are improvements, made as a result of many challenges running these tests, to the output text